test/ppr-tests-manifest.json

```diff
@@ -79,7 +79,8 @@
       "test/e2e/**/*.test.{t,j}s{,x}",
       "test/integration/app-*/**/*.test.{t,j}s{,x}",
       "test/production/app-*/**/*.test.{t,j}s{,x}",
-      "test/development/app-*/**/*.test.{t,j}s{,x}"
+      "test/development/app-*/**/*.test.{t,j}s{,x}",
+      "test/development/acceptance-app/error-message-url.test.ts"
```

Comment:
while https://github.com/vercel/next.js/pull/75882 is in flight, I'm going to add things that I'm fixing out of that PR to the manifest so we can verify the tests are fixed. We'll remove these from here once https://github.com/vercel/next.js/pull/75882 is merged.